sdk/metric/config_test.go

```diff
@@ -192,3 +193,11 @@ func TestWithView(t *testing.T) {
 	)})
 	assert.Len(t, c.views, 2)
 }
+
+func TestWithExemplarFilter(t *testing.T) {
```

Comment:
Missing test for the environment varaiable support.

Reply:
Added: https://github.com/open-telemetry/opentelemetry-go/pull/5850/commits/020cf0fbe0c03e320e7cc4df0b4a406df2a0c5cf